test/e2e/e2e/tests.py

```diff
@@ -0,0 +1,132 @@
+# Copyright 2020 Cortex Labs, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import time
+from http import HTTPStatus
+from pathlib import Path
+from typing import List
+
+import cortex as cx
+import yaml
+
+from e2e.expectations import parse_expectations, assert_response_expectations
+from e2e.utils import (
+    apis_ready,
+    endpoint_ready,
+    request_prediction,
+    job_done,
+    request_batch_prediction,
+)
+
+TEST_APIS_DIR = Path(__file__).parent.parent.parent / "apis"
+
+
+def delete_apis(client: cx.Client, api_names: List[str]):
+    for name in api_names:
+        client.delete_api(name)
+
+
+def test_realtime_api(client: cx.Client, api: str, timeout: int = None):
+    api_dir = TEST_APIS_DIR / api
+    with open(str(api_dir / "cortex.yaml")) as f:
+        api_specs = yaml.safe_load(f)
+
+    expectations = None
+    expectations_file = api_dir / "expectations.yaml"
+    if expectations_file.exists():
+        expectations = parse_expectations(str(expectations_file))
+
+    api_names = [api_spec["name"] for api_spec in api_specs]
+    for api_spec in api_specs:
+        client.create_api(api_spec=api_spec, project_dir=api_dir)
```

Comment:
Since we only make a request to a single API, I think it might make sense to only deploy the first API as well (and not deploy the others in the file), what do you think?